bindings/python/include/aligator/python/fwd.hpp

```diff
@@ -1,5 +1,5 @@
 #pragma once
-
+#include <pinocchio/fwd.hpp>
```

Comment:
That's not an acceptable change, because Pinocchio is an optional dependency. You could add include guards but most likely including Pinocchio's fwd header is too coarse a solution (what was the problem here?)